content/authoring/tutorials/create-first-kata.mdx

```diff
@@ -32,175 +32,192 @@ To create a new kata, you need to earn the ["Create Kata" privilege](/gamificati
+- Try to come up with a **novel and original idea**. Not only does this entail crafting a unique theme or backstory, but would ideally also involve a new, interesting approach to a problem. There are already many kata that cover a range of popular topics, including:
+  - Basics for beginners, such as the Fibonacci sequence, FizzBuzz, factorials, etc.
+  - Simple map/filter/reduce operations on lists or arrays. Numerous kata exist which merely require iteration over a sequence while selecting/transforming some of its elements based on simple criteria.
+  - Calculating the result of a mathematical, physical, or chemical formula. Substituting values into a formula has also been done before.
+  - Simple graph search algorithms like path finding, BFS, DFS, etc. [have been covered sufficiently](https://www.codewars.com/collections/bfs-shitpost).
+- **[Search for keywords](https://www.codewars.com/kata/search/?q=&) to find out if someone has already done something similar.** Not only basic topics have been extensively covered ([Fibonacci numbers](https://www.codewars.com/kata/search/?q=fibonacci&), [factorials](https://www.codewars.com/kata/search/?q=factorial&&order_by=rank_id%20asc), [FizzBuzz](https://www.codewars.com/kata/search/?q=buzz&&order_by=rank_id%20asc)), even advanced topics such as [Peano](https://www.codewars.com/kata/search/?q=peano) and [Church](https://www.codewars.com/kata/search/?q=church) numbers have multiple similar kata dedicated to them, as the search results clearly show.
+- **Solve more kata.** Doing so will offer you a better overview of what kinds of kata already exist. Having solved your 13th Fibonacci problem will help you understand why duplicates are as unpopular as they are.
 
 ### Kata quality
 
-Even the best idea won't save the kata if it's implemented poorly. Lucky for you, there's a set of [quality guidelines](/authoring/guidelines/) which, when followed, will ensure that all pieces of your kata will be created properly. There's quite a lot of them and in the beginning they might seem a bit intimidating, so try to read through them and understand them, before you apply them to your kata. Remember that whenever you have any problems understanding something, you can always [ask for help](https://gitter.im/Codewars/codewars.com/kata-authoring-help).
+Even the best idea won't save a kata from the community's critical eye if it is implemented poorly. Fortunately, there is a set of [quality guidelines](/authoring/guidelines/) which, when followed properly, will ensure that all individual aspects of your kata will combine into a well-regarded experience. While it may seem like a lot to take in, do make a point of reading and understanding them so you can apply them to your kata. Remember that whenever any problems or questions arise, you can always [ask for help](https://gitter.im/Codewars/codewars.com/kata-authoring-help).
```

Comment:
```suggestion
Even the best idea can be ruined by poor execution. Fortunately, there is a set of [quality guidelines](/authoring/guidelines/) which, when followed properly, will ensure that all individual aspects of your kata will combine into a well-regarded experience. While it may seem like a lot to take in, do make a point of reading and understanding them, so you can apply them to your kata. Remember that whenever any problems or questions arise, you can always [ask for help](https://gitter.im/Codewars/codewars.com/kata-authoring-help).
```